src/main/resources/languages/languages_de_DE.properties

```diff
@@ -4,6 +4,9 @@ startLogin=ANMELDUNG \u00d6FFNEN
 username=Benutzername
 password=Passwort
 repeatPassword=Passwort wiederholen
+changePassword=Passwort \u00e4ndern
```

Comment:
UTF8 ?

Reply:
As far as I know resource bundles are ISO 8859-1 encoded. Characters other than that must be declared in escaped unicode as it has been done in this project before, as well.